plugin/evm/wrapped_block.go

```diff
@@ -260,6 +266,10 @@ func (b *wrappedBlock) verify(predicateContext *precompileconfig.PredicateContex
 	// bootstrapping only verifies blocks that have been canonically accepted by
 	// the network, these checks would be guaranteed to pass on a synced node.
 	if b.vm.bootstrapped.Get() {
+		if err := b.verifyIntrinsicGas(); err != nil {
+			return fmt.Errorf("failed to verify intrinsic gas: %w", err)
+		}
```

Comment:
(not directly relevant to this PR) We should move these to `semanticVerify` in another PR